packages/api/helper/nominationGsheetToDB.js

```diff
@@ -63,7 +63,9 @@ module.exports = function gsheetToDB() {
             dischargeDate: nomination[32],
             patientDiagnosis: nomination[34],
             grantRequestInfo: nomination[35],
-            amountRequestedCents: parseFloat(nomination[37].replace("$", "").replace(",", "") ) * 100,
+            amountRequestedCents: parseInt(
+              parseFloat(nomination[37].replace(/\$/g, '')) * 100
```

Comment:
Do we want this change? I seem to recall a recent PR to change this line to what it currently is in develop.

Reply:
Definitely did not mean to change that code Austin. I've changed it back and pushed the code.